src/MLIR/AST/Dialect/ControlFlow.hs

```diff
@@ -49,7 +49,7 @@ cond_br cond trueBlock trueArgs falseBlock falseArgs = do
     , opRegions = []
     , opSuccessors = [trueBlock, falseBlock]
     , opAttributes = namedAttribute "operand_segment_sizes" $
-                       DenseArrayAttr $
+                       DenseElementsAttr (VectorType [3] $ IntegerType Unsigned 32) $
```

Comment:
The type of this field changed to DenseArrayAttr, we could just signed one here. Currently only signless is a thing upstream, but signless, signed and unsigned is planned to be supported.